cmd/empire/server.go

```diff
@@ -108,5 +108,16 @@ func newAuthenticator(c *cli.Context, e *empire.Empire) auth.Authenticator {
 		)
 	}
 
+	// After the user is authenticated, check their GitHub Team membership.
+	if teamId := c.String(FlagGithubTeam); teamId != "" {
+		authorizer := githubauth.NewTeamAuthorizer(client)
+		authorizer.TeamId = teamId
+
+		log.Println("Adding GitHub Team authorizer with the following configuration:")
+		log.Println(fmt.Sprintf("  Team ID: %v ", teamId))
+
+		return auth.WithAuthorization(authenticator, authorizer)
```

Comment:
ah, i didn't realize this was on every API call, I thought it just happened once when the user logged in.

definitely makes sense to cache.